resources/prometheus/prometheus-rules.yaml

```diff
@@ -639,3 +716,96 @@ spec:
             summary: "There is a high risk of over-committing CPU resources on worker nodes in AZ {{ $labels.availability_zone }}."
             description: "During the last 5 minutes, the average CPU limit commitment on worker nodes in AZ {{ $labels.availability_zone }} was {{ $value | humanizePercentage }}. This is above the recommended threshold of 200%."
             sop_url: "https://gitlab.cee.redhat.com/stackrox/acs-cloud-service/runbooks/-/blob/master/sops/dp-027-cluster-scale-up.md"
+
+        - alert: Central latency error budget exhaustion for GRPC API - 90%
+          annotations:
+            message: "Latency error budget exhaustion for central's GRPC API. Current exhaustion: {{ $value | humanizePercentage }}."
+          expr: |
+            (central:grpc_server_handling_seconds:rate10m:p90:sample_count_filter28d * central:grpc_server_handling_seconds:rate10m:p90:error_budget_exhaustion28d) >= 0.9
+          labels:
+            service: central
+            namespace: "{{ $labels.namespace }}"
+            rhacs_instance_id: "{{ $labels.namespace }}"
```

Comment:
The instance id is not the same as the namespace. The namespace is `rhacs-{instance_id}`.